src/mlir/generator.cc

```diff
@@ -811,6 +811,8 @@ namespace mlir::verona
     // Create function
     auto funcTy = builder.getFunctionType(types, retTy);
     auto func = mlir::FuncOp::create(loc, name, funcTy);
+    // FIXME: This needs to be set correctly later
```

Comment:
What does 'correctly' mean here?  I think everything in Verona will have internal / private linkage except for the main entry point.

Reply:
That's what I meant by "correctly": confirming that internal is the correct visibility here. I'm assuming the main entry point isn't any user function, but the entry point of the executable, right?